numpy/f2py/_src_pyf.py

```diff
@@ -1,3 +1,4 @@
+import os
```

Comment:
Hmm, how did this work before?

Reply:
Perhaps `resolve_includes`() is usually not called.